### DISCUSSION TITLE
Defer - onError - retry?

### DISCUSSION THREAD

**Discussion by impactvelocity** [remix-run/react-router#10027] [General]
The defer flow is great.

However, when an error happens - I want to allow my users to "retry". If I use fetcher.load(pathname) it will hit the loader, but it doesn't retrigger the loading.

https://reactrouter.com/en/main/guides/deferred 

I am a bit confused. It seems this is the intended behaviour.

However, what is the solution here? hard refresh the page?

**Comment by shura-v** [comment]
Any news here? Seems like a must have feature. You can for example navigate to something within the same route (e.g. hash) thus triggering revalidation, but that sounds like an ultra hack 🤷‍♂️
